src/integrationtests/resources/com/aws/greengrass/integrationtests/ipc/metricsConfig.yaml

```diff
@@ -0,0 +1,26 @@
+---
+services:
+  aws.greengrass.Nucleus:
+    configuration:
+      runWithDefault:
+        posixUser: nobody
+        windowsUser: integ-tester
+      logging:
+        level: "DEBUG"
+  aws.greengrass.clientdevices.Auth:
+    configuration:
+      deviceGroups:
+        formatVersion: "2021-03-05"
+        definitions:
+          devices:
+            selectionRule: "*"
+            policyName: "allowPublishMetrics"
+        policies:
+          allowPublishMetrics:
+            policyStatement1:
+              statementDescription: "publish metrics"
+              operations: "*"
+              resources: "*"
```

Comment:
this isn't correct.

You're confusing CDA policy with IPC's access controls. 
https://docs.aws.amazon.com/greengrass/v2/developerguide/interprocess-communication.html#ipc-authorization-policies